src/Common/Types.ts

```diff
@@ -19,6 +19,7 @@ import { Placement } from 'tippy.js'
 import { ImageComment, ReleaseTag } from './ImageTags.Types'
 import { ACTION_STATE, DEPLOYMENT_WINDOW_TYPE, DockerConfigOverrideType, SortingOrder, TaskErrorObj } from '.'
 import { RegistryType } from '../Shared'
+import { TippyProps as TippyJSProps } from '@tippyjs/react'
```

Comment:
External import should be above

Reply:
moved props type to the folder of Tooltip (new folder)